src/modules/markets/actions/load-markets-info.js

```diff
@@ -21,3 +21,13 @@ export const loadMarketsInfo = (marketIDs, callback = logError) => (dispatch, ge
     callback(null, marketsData)
   })
 }
+
+export const loadMarketsInfoOnly = (marketIDs, callback = logError) => (dispatch, getState) => {
```

Comment:
Can combine this w/ `loadMarketsInfo`

Reply:
I created the only version because the loadMarktsInfo caused a cascade of other api calls that would just collect data not needed for transactions.